src/main/java/de/muenchen/isi/infrastructure/entity/enums/lookup/ArtBaulicheNutzung.java

```diff
@@ -11,13 +13,21 @@
  */
 @AllArgsConstructor
 public enum ArtBaulicheNutzung implements ILookup {
+    UNSPECIFIED(ILookup.UNSPECIFIED),
     WR("Reines Wohngebiet (WR)"),
     WA("Allgemeines Wohngebiet (WA)"),
     MU("Urbanes Gebiet (MU)"),
     MK("Kerngebiet (MK)"),
     MI("Mischgebiet (MI)"),
-    GE("Gewerbegebiet (GE)");
+    GE("Gewerbegebiet (GE)"),
+    INFO_FEHLT("Info fehlt");
 
     @Getter
     private final String bezeichnung;
+
+    public static List<ArtBaulicheNutzung> getArtBaulicheNutzungForBauvorhaben() {
+        final var withoutUnspecified = EnumUtils.getEnumList(ArtBaulicheNutzung.class);
+        withoutUnspecified.remove(ArtBaulicheNutzung.UNSPECIFIED);
```

Comment:
Wieso wird der eine Enum Wert entfernt?

Reply:
Im Bauvorhaben gibt es eine Mehrfachauswahl. Da ist es besser wenn in der Mehrfachauswahl nichts gewählt ist und der Wert "--- Keine Auswahl ---" gar nicht auswählbar ist.

Reply:
Das Baugebiet ist ein einfaches Dropdown und wenn noch nichts gewählt wurde, dann soll `--- Keine Auswahl ---` als vorbelegung drinnen sein.